scripts/build_example

```diff
@@ -6,17 +6,19 @@ source "${SCRIPTS_DIR}/common"
 
 language="rust"
 compilation_mode='fastbuild'
-docker_image=false
-while getopts "l:ide:h" opt; do
+docker_config=''
+while getopts "l:i:de:h" opt; do
   case "${opt}" in
     h)
-      echo -e "Usage: ${0} [-h] [-l rust|cpp] [-i] [-d] -e EXAMPLE
+      echo -e "Usage: ${0} [-h] [-l rust|cpp] [-i base|logless] [-d] -e EXAMPLE
 
 Build the given example Oak Application and client.
 
 Options:
   -e    Example application name (required)
-  -i    Package application in a Docker image
+  -i    Package application in a Docker image with one of the following server types:
```

Comment:
I guess if this is not specified, it does not package it as a Docker image at all? Perhaps clarify somewhere (basically this flag enables docker packaging, *and* specifies the version)